src/components/Search/Search.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MoviesList } from '../MoviesList';
+
+export class Search extends React.Component {
+    state = {
+      query: '',
+    };
+
+    render() {
+      const { query } = this.state;
+      const { movies } = this.props;
+
+      const visiableMovies = movies
+        .filter(movie => movie.title.toLowerCase()
+          .includes(query.toLowerCase())
+            || movie.description.toLowerCase()
+              .includes(query.toLowerCase()));
+
+      return (
+        <>
+          <div className="box">
+            <div className="field">
+              <label htmlFor="searchQuery" className="label">
+                Search movie
+              </label>
+
+              <div className="searchField">
+                <input
+                  type="text"
+                  id="searchQuery"
+                  className="input"
+                  placeholder="Type search word"
+                  onChange={({ target }) => {
+                    this.setState({
+                      query: target.value,
+                    });
+                  }}
+                />
+              </div>
+            </div>
+          </div>
+          <MoviesList movies={visiableMovies} />
```

Comment:
if you  want to complete 4's task you need to keep this component in the App component